Gemfile

```diff
@@ -7,6 +7,7 @@ gem "administrate-field-image"
 gem "faker"
 gem "front_matter_parser"
 gem "globalid"
+gem "image_processing"
```

Comment:
Does this mean we'll now have a system dependency on `ImageMagick` or `libvips`?

I guess if you're doing this with Rails that's going to happen anyway, so maybe it's not a concern for us?

Reply:
I was going to see if it made sense to not require this here, and then the Rails app that is using Administrate would require it instead? So then if you are using Administrate but not ActionTest/ActiveStorage you won't be unnecessarily requiring this. Would that work?

Reply:
Ah, hm! Yeah, I'd marked it as resolved on the assumption it'd be okay but maybe it's not?

If you can, I think this is worth exploring.